tpgtools/templates/resource.html.markdown.tmpl

```diff
@@ -10,23 +10,22 @@
 	distributed under the License is distributed on an "AS IS" BASIS,
 	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 	See the License for the specific language governing permissions and
-	limitations under the License. */}}
-// ----------------------------------------------------------------------------
-//
-//     ***     AUTO GENERATED CODE    ***    Type: DCL     ***
-//
-// ----------------------------------------------------------------------------
-//
-//     This file is managed by Magic Modules (https://github.com/GoogleCloudPlatform/magic-modules)
-//     and is based on the DCL (https://github.com/GoogleCloudPlatform/declarative-resource-client-library).
-//     Changes will need to be made to the DCL or Magic Modules instead of here.
-//
-//     We are not currently able to accept contributions to this file. If changes
-//     are required, please file an issue at https://github.com/hashicorp/terraform-provider-google/issues/new/choose
-//
-// ----------------------------------------------------------------------------
-
+	limitations under the License. */ -}}
 ---
```

Comment:
Does this need to be above the comment?

Reply:
the `-}}` prevents it from making a newline

Reply:
consistent with mmv1